app/src/kv_types.h

```diff
@@ -96,6 +97,22 @@ namespace scitt
             std::string(JOSE_ALGORITHM_EDDSA)};
         }
       }
+      
+      bool is_accepted_issuers(std::string issuer) const
```

Comment:
```suggestion
      bool is_accepted_issuers(std::string_view issuer) const
```

Using an std::string_view here avoids an unnecessary copy.